### ISSUE TITLE
Installed in MSFS - not working properly/at all

### ISSUE THREAD

**Issue by JimmyDem** [Nukem9/dlssg-to-fsr3#201]
Good evening, I've followed the steps to installations in several videos into MSFS. I've tried the 0.5, 0.6, and 0.7 builds with no success. I know the mod works because in the settings, I get the option to select the frame generation. It unfortunately does not add any frames to the game. I have loaded up in LAX (One of the heaviest airports) and got 34-35 fps with it both enabled and disabled. It seems to work for this creator in his video: [https://www.youtube.com/watch?v=9VLqICEFUTI]

**System Sepcs:**
RTX 3080
i9-10900k
48gb RAM DDR4
2-165Hz monitors
Win 11

**Comment by lamiska** [comment]
HAGS enabled? DX12 in game enabled? Vsync in game disabled?

**Comment by JimmyDem** [comment]
Hi yes all are true! I only have V-Sync enabled in NVIDIA Control panel

**Comment by ForthtuN** [comment]
You are not using the in-game fps overlay from sdk are you?

**Comment by JimmyDem** [comment]
It was initially off;however, I later turned it on for the fps counter.
I’ve been playing with it on and off with no success.

**Comment by ForthtuN** [comment]
As I said, don't use it, it doesen't show the correct fps, use rtss or nvidias

**Comment by JimmyDem** [comment]
Yes, I just switched to it! It works but I often have this problem of micro
stuttering across the screen when I pan the view now. I set TAA to 100
